lang/po/de.po

```diff
@@ -25128,9 +25139,9 @@ msgstr "Stechapfel"
 
 #: lang/json/COMESTIBLE_from_json.py
 msgid "datura seeds"
-msgid_plural "datura seeds"
-msgstr[0] "Stechapfelsamen"
-msgstr[1] "Stechapfelsamen"
+msgid_plural "datura seedss"
```

Comment:
Fix for this issue is already landed, so this error in translation files will be removed in the next launch of extractor script.